DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Delegates/DelegateGroups/GroupDelegatesRemoveViewModelTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.TrackingSystem.Delegates.DelegateGroups
+{
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class GroupDelegatesRemoveViewModelTests
+    {
+        [Test]
+        public void GroupDelegatesRemoveViewModel_populates_expected_values()

```

Comment:
Is there much of a purpose to this test? I can't recall anywhere else we are testing down to this basic level of value assignment.

Reply:
Ah I'd written it before I wrote the helpers in the same commit. Made sense to test the string logic on the names before there were helpers. Will remove it now as there are unit tests for the helper.